scalafmt-tests/src/test/scala/org/scalafmt/CommentTest.scala

```diff
@@ -124,4 +130,19 @@ class CommentTest extends FunSuite {
     val obtained = Scalafmt.format(original, javadocStyle).get
     assertNoDiff(obtained, expected)
   }
+
+  test("preserve wikidoc headings when docstring wrap is enabled") {
+    val original = """
+      |/** =Heading1=
+      |  *
+      |  * ==Heading2==
+      |  *
+      |  * ===Heading3===
+      |  */
+      |""".stripMargin
+
+    val expected = original
```

Comment:
perhaps i'd request original to be different so that it's clear the formatter actually does something. that's why your `expected = original` looks out of place. :)

Reply:
That's a fair point, but I'm wondering if you have any ideas that would be relevant to this change? That's what I got stuck on when drafting the test.

So... since this depends on wrap being enabled, I should do things that'll get impacted by wrap and show that the headings are preserved even under those conditions.

Thanks, I'll hopefully have this PR updated tomorrow.

Reply:
Actually I was thinking about it, so just took a swag at this tonight. Happy to adjust it more (or draft some entirely different test altogether) if this update is insufficient.

Reply:
patrick, i am sorry, i don't know why i blanked out. this file `CommentTest` is specially crafted for comments and trailing space, something that cannot be done otherwise.

can you please put your test in `scalafmt-tests/src/test/resources/test/JavaDoc.stat` instead? this is where all the "normal" tests reside.

Reply:
Can-do -- pushed! That section of the documentation kinda confused me so thanks for the pointer to the correct file to place the test.